### PR TITLE
test_time2_variables

### PR DESCRIPTION
**Pull request by rohit-rk**

### CODE REVIEW

File: jx_sqlite/query_table.py [hunk 1]
```diff
@@ -229,12 +229,16 @@ def query(self, query):
                     allowNulls=allowNulls,
                     domain=domain
                 ))
+                
+            zeros = []
```
Comment (klahnakoski) [inline jx_sqlite/query_table.py]: zeros is never used, so we do not need it

File: jx_sqlite/query_table.py [hunk 2]
```diff
@@ -229,12 +229,16 @@ def query(self, query):
...
+            data_cubes = {}
+            for si, s in enumerate(listwrap(query.select)):
+                if s.aggregate == "count" and index_to_columns[si].push_child == ".":
+                    zeros.append(0)
```
Comment (klahnakoski) [inline jx_sqlite/query_table.py]: the old version had `len(zeros)==len(listwrap(query.select))`; nothing is appended to `zeros` in the `else` block to maintain that identity.

File: jx_sqlite/query_table.py [hunk 3]
```diff
@@ -229,12 +229,16 @@ def query(self, query):
...
+                    data_cubes[s.name] = Matrix(dims=dims, zeros=zeros[si])
```
Comment (klahnakoski) [inline jx_sqlite/query_table.py]: I believe `zeros[si]` is now `0`

**Comment by klahnakoski** [comment]
there is also `zeros` logic [at line 173](https://github.com/mozilla/jx-sqlite/pull/32/files#diff-713bf75df1f07d327115d355fdadc9bdR173).  be sure the same change is applied there, and it works.

File: jx_sqlite/query_table.py (hunk 3 above)
Comment (klahnakoski) [inline jx_sqlite/query_table.py]: ` and index_to_columns[si].push_child == "."` seems like an odd condition; count of anything can never be `None` , just zero, or more.  Do the tests pass with this removed?

Comment (rohit-rk) [inline jx_sqlite/query_table.py]: yes, that's correct. removed this and test passes.

**Comment by rohit-rk** [comment]
test passes for all the above mentioned changes.

**Pull request merged**